src/test/scala/chiselTests/ModuleChoiceSpec.scala

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.choice.{Case, Group, ModuleChoice}
+import chiselTests.{ChiselFlatSpec, MatchesAndOmits, Utils}
+import _root_.circt.stage.ChiselStage
+
+object Platform extends Group {
+  object FPGA extends Case
+  object ASIC extends Case
+}
+
+class TargetIO(width: Int) extends Bundle {
+  val in = Flipped(UInt(width.W))
+  val out = UInt(width.W)
+}
+
+class FPGATarget extends FixedIOExtModule[TargetIO](new TargetIO(8)) {}
+
+class ASICTarget extends FixedIOExtModule[TargetIO](new TargetIO(8)) {}
+
+class VerifTarget extends FixedIORawModule[TargetIO](new TargetIO(8)) {}
```

Comment:
```suggestion
class FPGATarget extends FixedIOExtModule[TargetIO](new TargetIO(8))

class ASICTarget extends FixedIOExtModule[TargetIO](new TargetIO(8))

class VerifTarget extends FixedIORawModule[TargetIO](new TargetIO(8))
```
Style nit, also same comment about not needing to specify the type parameter (but of course you can).